addon/services/raven.js

```diff
@@ -156,6 +156,23 @@ export default Service.extend({
     }
     return true;
   },
+  
+  /**
+   * Adds a message to the Raven breadcrumb trail, which will be sent
+   * when any call to captureMessage or captureException is made.
+   *
+   * @method captureBreadcrumb
+   * @param  {object} crumb The object representing the breadcrumb to capture
+   * @return {Boolean}
+   */
+  captureBreadcrumb(crumb) {
+    if (this.get('isRavenUsable')) {
+      Raven.captureBreadcrumb(crumb);
+    } else {
+      Ember.debug(crumb.message || crumb);
```

Comment:
At least in Firefox this just logs `[Object object]`, I'm not sure how to stop this.